junit-launcher/src/main/java/org/junit/gen5/launcher/main/Root.java

```diff
@@ -44,15 +50,13 @@ TestDescriptor getTestDescriptorFor(TestEngine testEngine) {
 	}
 
 	void applyPostDiscoveryFilters(TestDiscoveryRequest discoveryRequest) {
-		TestDescriptor.Visitor filteringVisitor = (descriptor, remove) -> {
-			if (!descriptor.isTest())
-				return;
-
-			if (isExcluded(discoveryRequest, descriptor)) {
+		Filter<TestDescriptor> postDiscoveryFilter = composeFilters(discoveryRequest.getPostDiscoveryFilters());
```

Comment:
Column width for Java source code is currently 120: https://github.com/junit-team/junit5/blob/master/src/eclipse/junit-eclipse-formatter-settings.xml#L291

Yes, this is enforced by the build via Spotless.